build/build.mk

```diff
@@ -30,6 +30,7 @@ export GO111MODULE := on
 export PATH := $(PROJECT_WORKSPACE)/bin:$(PATH)
 
 GO_TEST ?= test $(TEST_OPTIONS) $(TEST_FLAGS) $(SOURCE_FILES) -run $(TEST_PATTERN) -timeout=10m
+GO_TEST_SLOW ?= test $(TEST_OPTIONS) --tags=slow $(TEST_FLAGS) $(SOURCE_FILES) -run $(TEST_PATTERN) -timeout=10m
```

Comment:
I think this will run all tests without tags + tests with slow tag.
I think we should find a solution to not run the tests multiple times. Can you please check this?

Reply:
one possible solution is to mark ALL tests with some tags, but it will require much work. Let's add slow tests for the main unit tests job and see if it is acceptable.